lib/concurrent/edge/atomic_markable_reference.rb

```diff
@@ -0,0 +1,179 @@
+module Concurrent
+  module Edge
+    # @!macro atomic_markable_reference
+    class AtomicMarkableReference < ::Concurrent::Synchronization::Object
+      # @!macro [attach] atomic_markable_reference_method_initialize
+      def initialize(value = nil, mark = false)
+        super
```

Comment:
I think you do not need to wait for the fix to land in master, it can be merged earlier. The currently failing job is something else.